plugin/core/transports.py

```diff
@@ -220,27 +245,37 @@ def _stderr_loop(self) -> None:
 
 
 # Can be a singleton since it doesn't hold any state.
-json_rpc_processor = JsonRpcProcessor()
+standard_processor = StandardProcessor()
+node_ipc_processor = NodeIpcProcessor()
 
 
 def create_transport(config: TransportConfig, cwd: Optional[str],
                      callback_object: TransportCallbacks) -> Transport[Dict[str, Any]]:
+    stderr = subprocess.PIPE
+    # https://github.com/python/cpython/blob/17bf6b4671ec02d80ad29b278639d5307baddeb5/Lib/subprocess.py#L706
+    close_fds = True if sys.version_info >= (3, 8, 0) else subprocess._PLATFORM_DEFAULT_CLOSE_FDS  # type: ignore
```

Comment:
`_PLATFORM_DEFAULT_CLOSE_FDS` doesn't sound like a public API, judging from the leading underscore

It looks like just omitting that argument in the subprocess constructor would be the way to go?